src/main/resources/camunda/discontinue_claim_claimant.bpmn

```diff
@@ -130,6 +130,18 @@
       <bpmn:outgoing>Flow_03itfv5</bpmn:outgoing>
     </bpmn:serviceTask>
     <bpmn:sequenceFlow id="Flow_03itfv5" sourceRef="CreateDefendantDashboardNotificationsForDiscontinuance" targetRef="NotifyDiscontinuanceClaimant" />
+    <bpmn:exclusiveGateway id="Gateway_0ooegq0" name="Welsh party/docs preference?">
+      <bpmn:incoming>Flow_1n0vjnn</bpmn:incoming>
+      <bpmn:outgoing>Flow_0434mcf</bpmn:outgoing>
+      <bpmn:outgoing>Flow_04att3c</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:sequenceFlow id="Flow_1n0vjnn" sourceRef="GenerateNoticeOfDiscontinuance" targetRef="Gateway_0ooegq0" />
+    <bpmn:sequenceFlow id="Flow_0434mcf" name="No" sourceRef="Gateway_0ooegq0" targetRef="Gateway_00bae9b">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${empty WELSH_ENABLED || WELSH_ENABLED==false}</bpmn:conditionExpression>
+    </bpmn:sequenceFlow>
+    <bpmn:sequenceFlow id="Flow_04att3c" name="Yes" sourceRef="Gateway_0ooegq0" targetRef="Activity_0wretog">
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${WELSH_ENABLED==true}</bpmn:conditionExpression>
```

Comment:
If 'Welsh enabled' but there is no welsh party or doc lang is English.. still the camunda flow will be paused.  Need changes.